crates/db/src/queries.rs

```diff
@@ -115,44 +115,64 @@ pub(super) fn erc20_update_balance<'a>(
     .bind(balance.to_string())
 }
 
+// TODO - refactor
 pub(super) fn erc721_transfer<'a>(tx: &events::ERC721Transfer, chain_id: u32) -> Query<'a> {
     if tx.to.is_zero() {
         // burning
         sqlx::query(
-            r#" DELETE FROM nft_tokens WHERE chain_id = ? AND contract = ? AND token_id = ? "#,
+            r#" DELETE FROM erc721_tokens WHERE chain_id = ? AND contract = ? AND token_id = ? "#,
         )
         .bind(chain_id)
         .bind(format!("0x{:x}", tx.contract))
         .bind(format!("0x{:x}", tx.token_id))
     } else {
         // minting or transfer
         sqlx::query(
-            r#" INSERT OR REPLACE INTO nft_tokens (chain_id, contract, token_id, owner)
-                        VALUES (?,?,?,?)"#,
+            r#" INSERT OR REPLACE INTO erc721_tokens (contract, chain_id, token_id, owner, uri, metadata)
+                        VALUES (?,?,?,?, null, null)"#,
```

Comment:
if you're inserting null, why not just remove the `uri, metadata` part from the list of fields?